src/mds/Server.cc

```diff
@@ -799,7 +798,11 @@ void Server::handle_client_session(const cref_t<MClientSession> &m)
     break;
 
   default:
-    ceph_abort();
+    auto reply = make_message<MClientSession>(CEPH_SESSION_REJECT);
+    mds->send_message_client(reply, session);
```

Comment:
give a reason in this message. grep `send_reject_message` in this source file.

Reply:
`send_reject_message` is part of `case CEPH_SESSION_REQUEST_OPEN` while this is the `default` case, I can't use it here. Although, this also does the same thing, it makes a msg using `make_message` and then sends it, the only difference is `if (session->info.has_feature(CEPHFS_FEATURE_MIMIC))` which `send_reject_message` checks for.

Reply:
moved the functions from the scope of `case CEPH_SESSION_REQUEST_OPEN` to the scope of `handle_client_session` so that it can be used at other places within `handle_client_session`